auslib/test/test_db.py

```diff
@@ -4955,6 +4963,23 @@ def _add_jaws_test(self, db, upgrade=True):
             for table_name in base_jaws_tables:
                 self.assertNotIn("base_jaws", metadata.tables[table_name].c)
 
+    def _add_emergency_shutoff_tables(self, db, upgrade=True):
+        metadata = self._get_reflected_metadata(db)
+        shutoff_tables = ["emergency_shutoffs",
+                          "emergency_shutoffs_history",
+                          "emergency_shutoffs_scheduled_changes",
+                          "emergency_shutoffs_scheduled_changes_history",
+                          "emergency_shutoffs_scheduled_changes_conditions",
+                          "emergency_shutoffs_scheduled_changes_conditions_history",
+                          "emergency_shutoffs_scheduled_changes_signoffs",
+                          "emergency_shutoffs_scheduled_changes_signoffs_history"]
```

Comment:
I know it's a pain, but can you also do column-level verification here? One of the main things we try to catch here is column difference betweens db.py and the migration files.

Reply:
`testModelIsSameAsRepository` tests the column metadata, and generates a log like bellow:

```
E           AssertionError: Schema diffs:
E             table with differences: emergency_shutoffs_scheduled_changes
E               column with differences: complete
E                    model: Column('complete', Boolean(create_constraint=True, name=None), table=<emergency_shutoffs_scheduled_changes>, default=ColumnDefault(False))
E                 database: Column(u'complete', INTEGER(), table=<emergency_shutoffs_scheduled_changes>)

auslib/test/test_db.py:4824: AssertionError
```

Is this the same thing or you mean other test?

Reply:
Oh, heh. Sorry! I forgot how these tests worked while reviewing.